src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -88,71 +87,69 @@ export async function listItemGetController(
+          ...updatedAddressFields,
+        };
+      }
     }
-  }
 
-  if (listItem.status === "EDITED" || listItem.status === "OUT_WITH_PROVIDER") {
-    const eventForRequestedChanges = listItem?.history
-      ?.filter((event) => event.type === "OUT_WITH_PROVIDER")
-      .sort((a, b) => a.id - b.id)
-      .pop();
+    if (listItem.status === "EDITED" || listItem.status === "OUT_WITH_PROVIDER") {
+      const eventForRequestedChanges = listItem?.history
+        ?.filter((event) => event.type === "OUT_WITH_PROVIDER")
+        .sort((a, b) => a.id - b.id)
+        .pop();
 
-    requestedChanges = eventForRequestedChanges?.jsonData?.requestedChanges;
-  }
+      requestedChanges = eventForRequestedChanges?.jsonData?.requestedChanges;
+    }
+    const actionButtons: { [key: string]: string[] } = {
+      NEW: ["publish", "request-changes", "remove"],
+      OUT_WITH_PROVIDER: ["publish", "request-changes", "remove"],
+      EDITED: ["update", "request-changes", "remove"],
+      // ANNUAL_REVIEW: ["update", "request-changes", "remove"],
+      // REVIEW_OVERDUE: ["update", "request-changes", "remove"],
+      // REVIEWED: ["update", "request-changes", "remove"],
+      PUBLISHED: ["unpublish", "remove"],
+      UNPUBLISHED: ["publish", "request-changes", "remove"],
+    };
 
-  const actionButtons: { [key: string]: string[] } = {
-    NEW: ["publish", "request-changes", "remove"],
-    OUT_WITH_PROVIDER: ["publish", "request-changes", "remove"],
-    EDITED: ["update", "request-changes", "remove"],
-    // ANNUAL_REVIEW: ["update", "request-changes", "remove"],
-    // REVIEW_OVERDUE: ["update", "request-changes", "remove"],
-    // REVIEWED: ["update", "request-changes", "remove"],
-    PUBLISHED: ["unpublish", "remove"],
-    UNPUBLISHED: ["publish", "request-changes", "remove"],
-  };
-
-  const isPinned =
+    const isPinned =
     listItem?.pinnedBy?.some((user) => userId === user.id) ?? false;
-  const actionButtonsForStatus = actionButtons[listItem.status];
-
-  res.render("dashboard/lists-item", {
-    ...DEFAULT_VIEW_PROPS,
-    changeMessage: req.session?.changeMessage,
-    list,
-    listItem,
-    isPinned,
-    actionButtons: actionButtonsForStatus,
-    requestedChanges,
-    error,
-    title: serviceTypeDetailsHeading[listItem.type] ?? "Provider",
-    details: getDetailsViewModel(listItem),
-    csrfToken: getCSRFToken(req),
-  });
+    const actionButtonsForStatus = actionButtons[listItem.status];
+    res.render("dashboard/lists-item", {
+      ...DEFAULT_VIEW_PROPS,
+      changeMessage: req.session?.changeMessage,
+      list,
+      listItem,
+      isPinned,
+      actionButtons: actionButtonsForStatus,
+      requestedChanges,
+      error,
+      title: serviceTypeDetailsHeading[listItem.type] ?? "Provider",
+      details: getDetailsViewModel(listItem),
+      csrfToken: getCSRFToken(req),
+    });
+  } catch (error) {
+    logger.error(`listItemGetController Error: ${(error as Error).message}`);
```

Comment:
needs to bubble up as the error is swallowed up here - this is an anti-pattern.  Same advice as given above.